### ISSUE TITLE
Color themes/auto colors

### ISSUE THREAD

**Issue by thekeenant** [thekeenant/fcharts#16]
Colors must be manually specified for each element currently. It would be nice to have some sort of a theme object which colors refer to when they are not explicitly given a color. Any color could still be overridden, but the theme options object would be what developers would use to customize colors.

**Comment by thekeenant** [comment]
Theme options would have a color provider (i.e. monochromatic or analogous or random?). A line would refer to that provider using its index in its list as the parameter.

A stacked, grouped bar chart would be one scenario where both a primary provider and a secondary provider may be necessary. A single stack may be composed of similar bar colors (monochromatic), but stacks may differ from one another (analogous). Line charts would just use the primary provider.

**Comment by alexlovar** [comment]
hi, how i do to change the color of the line, in linechart?

**Comment by arthur109** [comment]
yeah, I cannot figure out how to change the color of the lines, but not the text or grid or scaffold, how would I do this?
